.github/workflows/bump_packages.yml

```diff
@@ -0,0 +1,94 @@
+name: Update packages
+on:
+  workflow_dispatch:
+env:
+  GIT_AUTHOR_NAME: ${{ github.actor }}
+  GIT_AUTHOR_EMAIL: ${{ github.actor }}@users.noreply.github.com
+
+jobs:
+  rpm_list:
+    name: 'Gather RPMs'
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-list.outputs.matrix }}
+    steps:
+    - name: Checkout RPM
+      uses: actions/checkout@v2
+      with:
+        ref: rpm/develop
+    - name: Set the list
+      id: set-list
+      run: ./list_updatable_packages
+
+  bump_rpm:
+    name: 'Bump ${{ matrix.package_name }} RPM ${{ matrix.new_version }}'
+    needs: rpm_list
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include: ${{ fromJson(needs.rpm_list.outputs.matrix) }}
+
+    steps:
+    - name: Install dependencies
+      run: |
+        sudo apt-get install -y rpm git-annex python3-semver
+        sudo gem install gem2rpm --no-document
+        sudo curl --create-dirs -o /usr/local/bin/spectool https://pagure.io/rpmdevtools/raw/26a8abc746fba9c0b32eb899b96c92841a37855a/f/spectool.in
+        echo 'echo "$GIT_AUTHOR_NAME <$GIT_AUTHOR_EMAIL>"' | sudo tee /usr/local/bin/rpmdev-packager
```

Comment:
Technically we can get `rpmdev-packager` from pagure in the same way as above, but that doesn't respect `GIT_AUTHOR_*` env variables.